lib/presentation/register/view/register_page_view.dart

```diff
@@ -0,0 +1,403 @@
+import 'package:book/models/app_user.dart';
+import 'package:book/styles/app_colors.dart';
+import 'package:book/styles/app_styles.dart';
+import 'package:book/validation/validation_form.dart';
+import 'package:flutter/cupertino.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_gen/gen_l10n/app_localizations.dart';
+
+class RegisterPageView extends StatefulWidget {
+  const RegisterPageView({super.key});
+
+  @override
+  State<RegisterPageView> createState() => _RegisterPageView();
+}
+
+class _RegisterPageView extends State<RegisterPageView> {
+  String? name;
+  String? lastName;
+  String? email;
+  String? password;
+  String? confirmPassword;
+  final _formKey = GlobalKey<FormState>();
+  late String message;
```

Comment:
when we use this variable?

Reply:
if you thought on String message that was previous version of code where message was errorMessage which we showed in snackBar after error